client-src/elements/chromedash-form-field.js

```diff
@@ -197,8 +197,9 @@ export class ChromedashFormField extends LitElement {
     const helpText = this.forEnterprise && this.fieldProps.enterprise_help_text ?
       this.fieldProps.enterprise_help_text :
       this.fieldProps.help_text;
-    const extraHelpText = this.fieldProps.extra_help;
-
+    const extraHelpText = this.forEnterprise ?
```

Comment:
Would it be better to show the regular extra_help if enterprise_extra_help is not defined?
Something like 
```
const extraHelpText = this.forEnterprise && 
     this.fieldProps.enterprise_extra_help ||
       this.fieldProps.extra_help;
```

Reply:
Hmm yes, I will do that. Updated the logic for the cases where we want to hide the help text for enterprises.